docs/source/getting_started/content/proc_configuring-the-virtual-machine.adoc

```diff
@@ -0,0 +1,50 @@
+[id="configuring-the-virtual-machine_{context}"]
+= Configuring the virtual machine
+
+The default number of vCPUs and amount of memory available to the {prod} virtual machine can be configured using the `cpus` and `memory` configurable properties, respectively.
+
+[IMPORTANT]
+====
+Changes to the configuration of the virtual machine require the creation of a new {prod} virtual machine.
+To create the new virtual machine, first delete the existing {prod} instance, then start a new instance with the configuration changes:
+
+[subs="+quotes,attributes"]
+----
+$ {bin} delete
+$ {bin} start
+----
+====
+
+[WARNING]
+====
+The [command]`{bin} delete` command will result in the loss of data stored in the {prod} virtual machine.
```

Comment:
"will result" => "results"

Unless there's a clear reason not to, use present tense. A clear reason means you can't avoid it. In this case, the present tense is clearer and less jargony.